src/soc/intel/common/tco.h

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#ifndef _INTELPCH_TCO_H_
+#define _INTELPCH_TCO_H_
+
+/* TCO registers and fields live behind TCOBASE I/O bar in SMBus device. */
+#define TCO_RLD				0x00
+#define TCO_DAT_IN			0x02
+#define TCO_DAT_OUT			0x03
+#define TCO1_STS			0x04
+#define  TCO1_STS_TCO_SLVSEL		(1 << 13)
+#define  TCO1_STS_CPUSERR		(1 << 12)
+#define  TCO1_STS_CPUSMI		(1 << 10)
+#define  TCO1_STS_CPUSCI		(1 << 9)
+#define  TCO1_STS_BIOSWR		(1 << 8)
+#define  TCO1_STS_NEWCENTURY		(1 << 7)
+#define  TCO1_STS_TIMEOUT		(1 << 3)
+#define  TCO1_STS_TCO_INT		(1 << 2)
+#define  TCO1_STS_OS_TCO_SMI		(1 << 1)
+#define  TCO1_STS_NMI2SMI		(1 << 0)
+#define TCO2_STS			0x06
+#define  TCO2_STS_SMLINK_SLAVE_SMI	(1 << 2)
+#define  TCO2_STS_SECOND_TO		(1 << 1)
+#define  TCO2_INTRD_DET			(1 << 0)
+#define TCO1_CNT			0x08
+#define  TCO1_LOCK			(1 << 12)
+#define  TCO1_TMR_HLT			(1 << 11)
+#define  TCO1_NMI2SMI_EN		(1 << 9)
+#define  TCO1_NMI_NOW			(1 << 8)
+#define TCO2_CNT			0x0A
```

Comment:
Some mixup in indentation?

Reply:
Single indentation for registers, double indentation for bitfields :)

Reply:
My changes are basically just these commits: https://github.com/Dasharo/coreboot/pull/472/files/78faf9f9c23ceaa09e24067b3b652535888e74b4..da615616e7459def116a2daecb63e8a16f0b4b0f

Reply:
Got confused by seeing this, thanks for clarification :upside_down_face: